eduNEXT/eox-tenant
# perf: added palm support and changed default settings to always use e…

This PR mainly updates the requirements for Palm support, and it modifies the setting USE_EOX_TENANT = True as I beleive we should have this as a default if eox_tenant is being installed and ready to use.

## Testing instructions

1. Install this version of eox-tenant in a palm environment.
2. Create routes and tenant configs accordingly as a super user
3. For example:
![Screenshot from 2023-11-16 20-36-29](https://github.com/eduNEXT/eox-tenant/assets/79876430/7cb01306-6196-47cc-99a7-b46af9662c58)
4. If you go to the specified route you should read the "site1" platform name.
5. Other tests cases can be found at: https://docs.google.com/document/d/1921hOajxAYrdin_thEZ005aQBwPNoj2zsbgkf3q1Mjc/edit
